sc-kpm/CMakeLists.txt

```diff
@@ -32,3 +32,5 @@ add_dependencies(sc-kpm
     sc-search
     sc-utils
 )
+
+SET(CMAKE_BUILD_TYPE Debug)
```

Comment:
Please don't commit this line for CMake config. This line should be added locally